### ISSUE TITLE
->= Doesn't render correctly (used in iOS vfl strings)

### ISSUE THREAD

**Issue by maxwellainatchi** [tonsky/FiraCode#414]
When typing "->=0-" in Apple's VFL, I expect it to look like "-≥0-", but it looks like "→=0-" which is very confusing

**Comment by dhouck** [comment]
I can see this going either way for different languages. Does anybody use anything which would make more sense with this rendered with the arrow?

**Comment by long-lazuli** [comment]
I think your editor syntax should priorize >= over -> , that's not something the font shoud manage, I guess...

**Comment by tonsky** [comment]
The best I can do is to turn off ligatures at all when `->` is followed by `=`. What is the correct way to parse this sequence anyways? I’m not familiar with VFL

**Comment by dhouck** [comment]
VFL is a [diagram-like language](https://developer.apple.com/library/content/documentation/UserExperience/Conceptual/AutolayoutPG/VisualFormatLanguage.html) and can have expressions surrounded by dashes; as the OP says, the correct parsing is `-≥0-`

**Comment by maxwellainatchi** [comment]
@dhouck Yeah, that's what that means. 
@tonsky Is there no way you could make it so that the ligature for `->=` just renders as `-≥`? I can't imagine any situation that you would need to put an arrow immediately before an equals sign, especially without a space in between.

**Comment by tonsky** [comment]
@maxwellainatchi yeah it can be done. I just want to make sure that we won’t break any other use-case
